configure.ac

```diff
@@ -1566,6 +1566,21 @@ AM_CONDITIONAL([OPENTHREAD_EXAMPLES_NRF52840], [test "${with_examples}" = "nrf52
 AC_SUBST(OPENTHREAD_EXAMPLES_SAMR21)
 AM_CONDITIONAL([OPENTHREAD_EXAMPLES_SAMR21], [test "${with_examples}" = "samr21"])
 
+#
+# libstdc++
+#
+AC_ARG_WITH(
+    [libstdcxx],
+    [AS_HELP_STRING([--with-libstdcxx], [Specify the C++ standard library to link to @<:@default=none@:>@.])],
```

Comment:
Is it useful to have this configure option?

Reply:
It seems there may be different names for c++ run time library on different system. For example, https://developer.android.com/ndk/guides/cpp-support#libc